internal/commands/result.go

```diff
@@ -362,21 +362,26 @@ func SummaryReport(
 		if err != nil {
 			return nil, err
 		}
-
 		summary.APISecurity = *apiSecRisks
 	}
 
 	for _, result := range results.Results {
 		countResult(summary, result)
 	}
 	if summary.SastIssues == 0 {
-		summary.SastIssues = notAvailableNumber
+		if !contains(summary.EnginesEnabled, commonParams.SastType) {
+			summary.SastIssues = notAvailableNumber
+		}
 	}
 	if summary.ScaIssues == 0 {
-		summary.ScaIssues = notAvailableNumber
+		if !contains(summary.EnginesEnabled, scaType) {
+			summary.ScaIssues = notAvailableNumber
+		}
 	}
 	if summary.KicsIssues == 0 {
-		summary.KicsIssues = notAvailableNumber
+		if !contains(summary.EnginesEnabled, commonParams.KicsType) {
```

Comment:
Can you join the if conditions as an && in the enclosing if? Ty

Reply:
Same for SCA and SAST

Reply:
Thanks for remembering! :)